planetiler-custommap/src/main/java/com/onthegomap/planetiler/custommap/ConfiguredFeature.java

```diff
@@ -84,6 +84,7 @@ public ConfiguredFeature(String layer, TagValueProducer tagValueProducer, Featur
     }
     processors.add(makeFeatureProcessor(feature.minZoom(), Integer.class, Feature::setMinZoom));
     processors.add(makeFeatureProcessor(feature.maxZoom(), Integer.class, Feature::setMaxZoom));
+    processors.add(makeFeatureProcessor(feature.minPixelSize(), Double.class, Feature::setMinPixelSize));
```

Comment:
I think we want some kind of test for this in `ConfiguredFeatureTest` to make sure it's wired-up properly.